gaia-list.js

```diff
@@ -81,15 +85,13 @@ module.exports = component.register('gaia-list', {
     els.container.appendChild(els.ripple);
     this.els.inner.appendChild(els.container);
 
-    // var reflow = els.ripple.offsetTop;
-    var scale = pos.item.width / 1.2;
     var duration = 500;
 
-    setTimeout(function() {
+    requestAnimationFrame(function() {
```

Comment:
Ouch! You are right. update PR only called requestAnimationFrame only before the function.